run_sim.py

```diff
@@ -6,27 +7,37 @@
 from time import sleep
 from StatsParser import StatsParser
 from StatsAnalysis import *
+import pexpect
 
 SIM_RUN_TIME = 30
 OUTPUT_DIR = "./out/"
 BUILD = "./Debug"
 SIM_LOGGING_TIME = 5
+KEY = "fill_in_pass"
 
 MPI_IMPL = 'OMPI'
 
 def kill_sim(process):
     sleep(SIM_RUN_TIME)
-    if (process.poll() is None):
-        print "sending SIGUSR1"
-        process.send_signal(signal.SIGUSR1)
+    if (process.isalive()):
+        print "[info] sending " + str(signal.SIGUSR1) + " to pid: " + str(process.pid)
+        process.kill(signal.SIGUSR1)
     sleep(SIM_LOGGING_TIME) # wait for the sim to finish up and write results
     print "killing process"
-    process.terminate()
-
-    # process.wait()
+    process.close()
+
+# def kill_sim(process):
```

Comment:
Can get rid off these old commented-out code blocks once we're confident in the new stuff.